### PR TITLE
Changing 202 responses content-type in the examples to xml/json instead ...

### PR DESCRIPTION
**Pull request by rnirmal**
...of plain/text

### CODE REVIEW

File: tests/examples/examples/local.conf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "directory": "/src/apidocs/src/resources/samples/",
+    "directory": "/opt/stack/reddwarf_lite-integration/apidocs/src/resources/samples/",
```
Comment (hub-cap) [inline tests/examples/examples/local.conf]: does this need changing?

**Comment by hub-cap** [comment]
+1

File: tests/examples/examples/local.conf (hunk 1 above)
Comment (rnirmal) [inline tests/examples/examples/local.conf]: probably not... that one just slipped. removing it.

**Comment by ed-** [comment]
It's very important that you know the mimetype of the empty strings you're getting.

+1

**Comment by imsplitbit** [comment]
+1

**Pull request merged**